### DISCUSSION TITLE
LDAP Troubleshooting

### DISCUSSION THREAD

**Discussion by phreakadelle** [kafbat/kafka-ui#470] [Q&A] [answered]
Hello Team, i am pretty lost on configuring LDAP connection for kafbat/kafka-ui. I can see my ldap.yml is used. However, the application is not providing any log or stacktrace if things go wrong. The only thing i can find in the application log is

```"09:57:55.932 [boundedElastic-1] DEBUG org.springframework.security.web.server.authentication.AuthenticationWebFilter - Authentication failed: Uncategorized exception occured during LDAP processing"```

I have started the app already with "--debug" to get more information but i cant find anything that tells me, what i am doing wrong. Would be happy if someone can point me into the right direction. Thanks

**Comment by Haarolean** [comment]
Hi, you can try starting with out example compose file designed for LDAP: https://github.com/kafbat/kafka-ui/blob/main/documentation/compose/ui-ldap.yaml

**Comment by phreakadelle** [comment, marked as answer]
Thanks @Haarolean  for the quick turn around. I was able to figure out the issue. I was able to figure it out by changing LDAP urls and make them incorrect. I could see the connection failed, whereas with the correct URL i just got the the "uncategorized exception".  I had to configure the javax.net.truststore to establish the connection now. Simple solution but hard to find. It would be much better if Spring or this App would have a more meaningful error, when there is something wrong with certs and missing trust.
